### PR TITLE
Classical shadow doc fix

### PR DESCRIPTION
**Pull request by rmoyard**

### CODE REVIEW

File: pennylane/shadows/transforms.py [hunk 1]
```diff
@@ -210,10 +210,5 @@ def circuit(x):
             [ 0.33705002-0.01125j   ,  0.011025  +0.232575j  ,
              -0.006075  +0.225225j  ,  0.33475   +0.j        ]],
            dtype=complex64, requires_grad=True)
-    >>> qml.jacobian(circuit)(x)
```
Comment (rmoyard) [inline pennylane/shadows/transforms.py:213]: Impossible to run this part with the jacobian, @eddddddy @Qottmann

File: pennylane/shadows/classical_shadow.py [hunk 2]
```diff
@@ -390,16 +391,17 @@ def max_entangled_circuit():
 
             @qml.qnode(dev)
             def qnode(x):
-                for i in range(n_wires):
+                for i in range(wires):
+
                     qml.RY(x[i], wires=i)
```
Comment (rmoyard) [inline pennylane/shadows/classical_shadow.py:396]: I cannot run this example as well because I don't have the right device and the array x, @eddddddy @Qottmann

File: pennylane/shadows/transforms.py (hunk 1 above)
Comment (eddddddy) [reply pennylane/shadows/transforms.py:213]: This line should be `qml.jacobian(lambda x: np.real(circuit(x)))(x)`, since autograd doesn't support complex gradients. 

You can keep the output below this. I think I just copied the previous line wrong.

File: pennylane/shadows/classical_shadow.py (hunk 2 above)
Comment (eddddddy) [reply pennylane/shadows/classical_shadow.py:396]: Not sure what values @Qottmann was using. If needed we can just redefine `x` and change the output below.

**Review by eddddddy — APPROVED** [review]
Looks good 💯
